docs/application/dotnet/guides/applications/uiapplication/ui-app.md

```diff
@@ -11,14 +13,14 @@ To create a basic UI application, you must:
     change events. You can override the [methods triggered for these
     events](#callback) to react to them.
 
-- Manage [application states and transitions](#state_trans) during the
+- Manage [application states and transitions](./application_lifecycle.md) during the
     application life-cycle.
+
 - Define a [background category](#allow_bg) for your application, if
     you want it to run in the background.
 
-
 <a name="callback"></a>
-## Event Handling
+## Event handling
```

Comment:
https://github.com/Samsung/tizen-docs/pull/1494#discussion_r783851602

I can't tell if this gerund case has been addressed or not.